src/components/Buildbaron/BBFIleTicket.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import { useMutation } from "@apollo/client";
+import styled from "@emotion/styled";
+import Button, { Variant } from "@leafygreen-ui/button";
+import { Popconfirm } from "antd";
+import {
+  BbCreateTicketMutation,
+  BbCreateTicketMutationVariables,
+} from "gql/generated/types";
+import { FILE_JIRA_TICKET } from "gql/mutations/file-jira-ticket";
+import { BBTitle } from "./BBComponents";
+
+export const BBFileTicket: React.FC<{
+  taskId: string;
+  dispatchBanner;
+  setCreatedTicketsCount;
+  createdTicketsCount;
```

Comment:
Can you properly type these fields

Reply:
Also we should utilize the same pattern for initializing types in components. So lets explicitly initialize the interface instead of doing it inline.